BOOST-708: add multiple aws profiles (#229)
boostercloud/booster: docs/README.md

aws_secret_access_key = <YOUR SECRET ACCESS KEY>
```

##### Multiple AWS Accounts

In case you are using multiple AWS accounts and not depending on the default profile, you will need to add a `region` field under the profile we will be using in the `credentials` file as seen on the right.

```ini
# ~/.aws/credentials
[default]
aws_access_key_id = <YOUR ACCESS KEY ID>
aws_secret_access_key = <YOUR SECRET ACCESS KEY>
region=<Region you want to deploy the application to>

[other_profile]
aws_access_key_id = <YOUR ACCESS KEY ID>
aws_secret_access_key = <YOUR SECRET ACCESS KEY>
region=<Region you want to deploy the application to>
```

AWS allows for a `config` file to be created, defining the region to deploy to, but this file will only work for the default profile.

> 

Reviewer: I'm not completely sure this is true. I have these 2 files:

```
# ~/.aws/config
[profile booster]
region=eu-west-3
output=table

# ~/.aws/credentials
[booster]
aws_access_key_id=foo
aws_secret_access_key=bar
```

And it's working for me. I have to `export AWS_PROFILE=booster` before running my `boost deploy` or `boost nuke` command

Author: Maybe it's because of the keyword `profile` in `[profile booster]`. When I was doing tests I was trying to have it as
```
# ~/.aws/config
[default]
region=us-west-1

[booster]
region=us-west-1
```

and it wasn't working for me. I'll try it out once more with your config and see if it works for me, if it does, then I'll have to update the block once again.

I'll try and remove the unrelated commits now, thanks

Author: You were right @itrion, I'll update the text to show users both ways of using the profiles and letting them know they have to export the profile in order for it to work.

aws_secret_access_key = <YOUR SECRET ACCESS KEY>
```

##### Multiple AWS Accounts

> In case you are using multiple AWS accounts and not depending on the default profile,

Reviewer 2: ```suggestion
If you are using multiple AWS accounts and don't want to use the default profile,
```

> you will need to add a `region` field to let AWS know which region we should deploy our application to.

Reviewer 2: ```suggestion
you will need to set a `region` option to let the AWS SDK know which region you want your application to be deployed to.
```

To do so, we have two possible solutions:

> * Adding the region to the profile in the `credentials` file

Reviewer 2: ```suggestion

* Add the region to the profile of your choice in your `~/.aws/credentials` file
```

```ini

> # ~/.aws/credentials
> [default]
> aws_access_key_id = <YOUR ACCESS KEY ID>
> aws_secret_access_key = <YOUR SECRET ACCESS KEY>
> region=<Region you want to deploy the application to>
> 
> [other_profile]
> aws_access_key_id = <YOUR ACCESS KEY ID>
> aws_secret_access_key = <YOUR SECRET ACCESS KEY>
> region=<Region you want to deploy the application to>

Reviewer 2: Let's be clearer, if we use the same placeholders in both profiles, some people might thing they need to set their Booster credentials in both places.
```suggestion
# ~/.aws/credentials
[default]
aws_access_key_id = <DEFAULT ACCESS KEY ID>
aws_secret_access_key = <DEFAULT SECRET ACCESS KEY>
region=<DEFAULT REGION>

[other_profile] # Give this profile the name that works best for you
aws_access_key_id = <YOUR ACCESS KEY ID>
aws_secret_access_key = <YOUR SECRET ACCESS KEY>
region=<REGION FOR YOUR BOOSTER APP>
```

```

> * Or creating a `config` file

Reviewer 2: ```suggestion

* Or creating a `~/.aws/config` file

```

```ini
# ~/.aws/config

> [default]
> region=<Region you want to deploy the application to>
> 
> [profile other_profile]
> region=<Region you want to deploy the application to

Reviewer 2: ```suggestion
[default]
region=<DEFAULT REGION>

[profile other_profile] # You can rename the profile in any way that works for you
region=<REGION FOR YOUR BOOSTER APP>
```
